inference-engine/src/auto_plugin/auto_plugin.cpp

```diff
@@ -153,7 +153,7 @@ std::vector<AutoPlugin::DeviceInformation> AutoInferencePlugin::GetDeviceChoice(
     auto getDeviceConfig = [&] (const DeviceName & deviceWithID) {
         IE::DeviceIDParser deviceParser(deviceWithID);
         std::string deviceName = deviceParser.getDeviceName();
-        ConfigType tconfig = mergeConfigs(_config, config);
```

Comment:
why don't we merge configs?

Reply:
because the config is already be merged.